project_event/views/activity_template_view.xml

```diff
@@ -40,6 +45,7 @@
                 <field name="event_template_id" string="Template Event"/>
                 <field name="temp_resp_id"/>
```

Comment:
@rimbendhaou: Refer the same comment in the model too for the variable name temp_resp_id. (Same comment applicable to other place and location)